zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1245,6 +1249,22 @@ impl<F: Field> SubCircuit<F> for PiCircuit<F> {
         let pi_cells = layouter.assign_region(
             || "region 0",
             |mut region| {
+                // Annotate columns
```

Comment:
I noticed that all the selectors will become fixed columns but we can't name them.  @CPerezz did you consider a `name_selector` method for the Region?

Reply:
Yes, it's challenging as Virtual Selectors get multiplexed. And so, annotate them is not trivial. So I decided not to do so.